rest-api-spec/src/main/resources/rest-api-spec/test/cat.nodes/10_basic.yml

```diff
@@ -58,6 +58,32 @@
       $body: |
                /^      http \n ((\d{1,3}\.){3}\d{1,3}:\d{1,5}\n)+  $/
 
+  - do:
```

Comment:
one last thing that I forgot. We'd need to get this new section into its own test e.g.

```
---
"Additional disk information":
```

This way we can add a skip section to it:

```
 - skip:
    version: " - 5.5.99"
    reason:  additional disk info added in 5.6.0
```